src/ui/Dropdown.tsx

```diff
@@ -68,11 +68,40 @@ type ItemProps = {
   children: ReactNode;
 };
 
-const Item = ({ href, onClick, children }: ItemProps) => (
-  <BootstrapDropdown.Item href={href} onClick={onClick}>
-    {children}
-  </BootstrapDropdown.Item>
-);
+const Item = ({ href, onClick, children }: ItemProps) => {
+  if (onClick) {
+    return (
+      <BootstrapDropdown.Item
+        as={Button}
+        variant={ButtonVariants.SECONDARY}
+        onClick={onClick}
+      >
+        {children}
+      </BootstrapDropdown.Item>
+    );
+  }
+
+  if (href) {
+    return (
+      <BootstrapDropdown.Item
+        forwardedAs={Link}
+        variant={LinkVariants.BUTTON_SECONDARY}
+        href={href}
+        padding={0}
+        css={`
+          .btn {
+            flex: 1;
+            border: none;
+          }
+        `}
+      >
+        {children}
+      </BootstrapDropdown.Item>
```

Comment:
What is the reason of doing it like this exactly? I don't really see the advantage, only some workarounds to make it look the same and a lot more code 😅

Reply:
Plus we should test if keyboard navigation inside the dropdown still works as expected now.

Reply:
For links this is needed, we want to use nuxt-link behind the scenes, so thath the page doesn't have to refresh when you click the url.

Reply:
the normal behaviour is an anchor tag = full page refresh

Reply:
Okay I see!